### PR TITLE
Add missing docs for COLLECT

### PR DESCRIPTION
**Pull request by Lojjs**
These parts existed for EXISTS and COUNT, but not for COLLECT.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-605.surge.sh

**Review by gem-neo4j — APPROVED** [review]
Yep, looks reasonable to me :)

File: modules/ROOT/pages/keyword-glossary.adoc [hunk 1]
```diff
@@ -1018,12 +1018,14 @@ xref::functions/temporal/index.adoc#functions-temporal-truncate-overview[Truncat
 | xref::syntax/expressions.adoc#query-syntax-case[CASE Expression]
 | A generic conditional expression, similar to if/else statements available in other languages.
 
-| xref:syntax/expressions.adoc#existential-subqueries[EXISTS {...}]
-| An `EXISTS` expression is used to evaluate the existence of a subquery.
+| xref:syntax/expressions.adoc#collect-subqueries[COLLECT {...}]
+| An expression used to create a list with the rows returned by a subquery.
```
Comment (renetapopova) [inline modules/ROOT/pages/keyword-glossary.adoc]:
```suggestion
| Creates a list with the rows returned by a subquery.

File: modules/ROOT/pages/keyword-glossary.adoc [hunk 2]
```diff
@@ -1018,12 +1018,14 @@ xref::functions/temporal/index.adoc#functions-temporal-truncate-overview[Truncat
...
 
 | xref:syntax/expressions.adoc#count-subqueries[COUNT {...}]
 | An expression used to compute the number of results of a subquery.
 
+| xref:syntax/expressions.adoc#existential-subqueries[EXISTS {...}]
+| An `EXISTS` expression is used to evaluate the existence of a subquery.
```
Comment (renetapopova) [inline modules/ROOT/pages/keyword-glossary.adoc]:
```suggestion
| Evaluates the existence of a subquery.
```

File: modules/ROOT/pages/keyword-glossary.adoc [hunk 3]
```diff
@@ -1018,12 +1018,14 @@ xref::functions/temporal/index.adoc#functions-temporal-truncate-overview[Truncat
 | xref::syntax/expressions.adoc#query-syntax-case[CASE Expression]
 | A generic conditional expression, similar to if/else statements available in other languages.
 
-| xref:syntax/expressions.adoc#existential-subqueries[EXISTS {...}]
-| An `EXISTS` expression is used to evaluate the existence of a subquery.
+| xref:syntax/expressions.adoc#collect-subqueries[COLLECT {...}]
+| An expression used to create a list with the rows returned by a subquery.
 
 | xref:syntax/expressions.adoc#count-subqueries[COUNT {...}]
 | An expression used to compute the number of results of a subquery.
```
Comment (renetapopova) [inline modules/ROOT/pages/keyword-glossary.adoc]:
```suggestion
| Computes the number of results of a subquery.
```

File: modules/ROOT/pages/keyword-glossary.adoc [hunk 4]
```diff
@@ -1018,12 +1018,14 @@ xref::functions/temporal/index.adoc#functions-temporal-truncate-overview[Truncat
 | xref::syntax/expressions.adoc#query-syntax-case[CASE Expression]
```
Comment (renetapopova) [inline modules/ROOT/pages/keyword-glossary.adoc]:
```suggestion
| xref::syntax/expressions.adoc#query-syntax-case[CASE]
```

**Review by renetapopova — APPROVED** [review]
Thanks, @Lojjs! I added some suggestions that simplify the descriptions and also made the coherent with the ones in the other tables. Otherwise, looks good.

**Pull request merged**